saturn_client/core.py

```diff
@@ -305,6 +305,21 @@ def get_jupyter_server(self, jupyter_server_id) -> Dict[str, Any]:
             ) from err
         return response.json()
 
+    def get_jupyter_token(self, jupyter_server_id) -> str:
```

Comment:
```suggestion
    def get_jupyter_token(self, jupyter_server_id: str) -> str:
```

Reply:
I copy-pasted from the _one_ method that was missing this :upside_down_face: